h5d.go

```diff
@@ -177,11 +177,71 @@ func (s *Dataset) OpenAttribute(name string) (*Attribute, error) {
 	return openAttribute(s.id, name)
 }
 
+//H5_DLL hid_t H5Dget_space(hid_t dset_id);
```

Comment:
Sure, that's what it is, but why is it here? There should be user-facing documentation (possibly, but by no means guaranteed including such a line) - think about how godoc renders this.